2264/2/versus.rb

```diff
@@ -0,0 +1,24 @@
+require 'optparse'
+require 'terminal-table'
+require_relative 'storage'
+require_relative 'rapper_data'
+require_relative 'rappers_list'
+require_relative 'table_printer'
+require_relative 'row_presenter'
+
+options = { 'top-bad-words' => nil }
+
+parser = OptionParser.new do |opts|
+  opts.on('--top_bad_words=number') do |number|
+    options['top-bad-words'] = number
+  end
+
+  opts.on('--help') do
+    puts 'Usage:'
+    puts 'Type --top-bad-words=N , there N - number of top rappers with bad fantasy'
+  end
+end
+
+parser.parse!
+
+TablePrinter.new(options['top-bad-words']) unless options['top-bad-words'].nil?
```

Comment:
it's not clear for me why TablePrinter parses rap battle. From it's name it is obvious it should print a table